datahost-ld-openapi/src/tpximpact/datahost/ldapi/handlers.clj

```diff
@@ -92,18 +93,16 @@
                         (matcha/index-triples)
                         (triples->ld-resource))]
     (if (= accept "text/csv")
-      (-> {:status 200
-           :headers {"content-type" "text/csv"
-                     "content-disposition" "attachment ; filename=release.csv"}
-           :body (let [change-infos (db/get-changes-info triplestore release-uri Integer/MAX_VALUE)]
-                   (if (empty? change-infos)
-                     "" ;TODO: return table header here, need to get schema first
-                     (let [key (:snapshotKey (last change-infos))]
-                       (when (nil? key)
-                         (throw (ex-info "No CSV snapshot for release found."
-                                         {:path-params path-params})))
-                       (ring-io/piped-input-stream (partial change-store-to-ring-io-writer change-store key)))))}
-          (shared/set-csvm-header request))
+      (let [change-infos (db/get-changes-info triplestore release-uri Integer/MAX_VALUE)]
+        (cond
+          (empty? change-infos) (-> (util.response/response "This release has no revisions yet")
+                                    (util.response/status 422)
+                                    (util.response/header "content-type" "text/plain"))
+          :else (let [rev-uri ^URI (-> change-infos last :rev)]
+                  (-> (.getPath rev-uri)
+                      (util.response/redirect)
+                      (util.response/header "accept" accept)
```

Comment:
I think we should remove this line setting an `accept` header. 

1. `accept` is in the wrong context... iirc it should be set on requests not responses (so would want to be `content-type` here)... though this is moot because...
2. we can leave it up to the redirected to set the appropriate content-type.

We should however set the `csvm-header` here though, as it will track the release over time.

Reply:
thanks, the "accept" obviously makes no sense.